rustup-init.sh

```diff
@@ -385,13 +385,21 @@ downloader() {
     else
         _dld='curl or wget' # to be used in error message of need_cmd
     fi
+	
+	if [ -n "$RUSTUP_USE_UNSAFE_SSL" ]; then
+		_curl_unsafe = "--insecure"
+		_wget_unsafe = "--no-check-certificate"
```

Comment:
Are these long options reliable for all implementations of wget?  E.g. does BSD have different arguments?  I'll be satisfied if you say "yes" but I'd like to ensure you've checked.

Reply:
AFAIK wget is a GNU program, not a POSIX command that is different on other UNIX-like OSes. NetBSD, of example, works with wget 1.17, and this option has been there since 1.10

Basically, the answer is yes.

Reply:
BusyBox uses a non-standard wget. For instance: the `-N` flag does not exist in BusyBox wget.